tests/playbooks/e2e.yaml

```diff
@@ -117,3 +120,21 @@
       with_items:
         - "stopped"
         - "running"
+
+    - name: Expose the VMI using service
+      k8s_service:
+        state: present
+        name: test-fedora-vmi
+        namespace: default
+        type: NodePort
+        ports:
+          - port: 20022
+            protocol: TCP
+        selector:
+          kubevirt.io/vm: test-working
```

Comment:
I do not see `test-working` anywhere. I see that everywhere we are using vm name as parameter ` '{{ vm_name }}'`. I think we should do it here as well.

Reply:
Yes, I did those changes need to push.